index.js

```diff
@@ -25,5 +25,12 @@ module.exports = {
    *
    * @param {object} options The AWS.S3 constructor options.
    */
-  getS3: require('./lib/getS3')
+  getS3: require('./lib/getS3'),
+
+  /**
+   * Returns a Promises compliant AWS.EC2 api.
+   *
+   * @param {object} options The AWS.EC2 constructor options.
+   */
+  getEC2: require('./lib/getEC2')
```

Comment:
I would like the properties of this object in alphabetical order. It's not a problem now. But there are a lot of AWS services and it will make it easier to find things.